### DISCUSSION TITLE
neper questions

### DISCUSSION THREAD

**Discussion by tage-123** [neperfepx/neper#880] [Q&A] [answered]
First I generate a Voronoi tessellation containing 100 cells with experimental grain-growth morphological
properties and apply regularization.Then mesh tessellation n100-id1.tess with a mesh size of rcl = 0.02 but error
![{A4AC4FAC-7065-435b-84CF-5ADB2CCB49F0}](https://github.com/neperfepx/neper/assets/174629615/0f2c9183-dc82-4364-a910-1ce35c4905c4)
![{1905145F-F2ED-411b-B5BB-07F0315C22D9}](https://github.com/neperfepx/neper/assets/174629615/b812168e-cd2d-42ae-a5c4-b891700d3f1c)

**Comment by rquey** [comment, marked as answer]
`-rcl 0.02` generates a huge number of elements, so, it might be it.
